### DISCUSSION TITLE
flink sql task

### DISCUSSION THREAD

**Discussion by xizhimen** [apache/dolphinscheduler#13677] [Q&A]
env : 
    dolphinsheduler 3.1.4 
    flink 1.14.2

problem:
    cannot run flink sql task
    
![image](https://user-images.githubusercontent.com/19748550/223029323-427d1640-3653-4a9e-8491-782b5714926b.png)
![image](https://user-images.githubusercontent.com/19748550/223029373-200ee6ea-73a7-44da-b0be-fdb397b59bdd.png)
 error log:
    
![image](https://user-images.githubusercontent.com/19748550/223029638-7e276634-11bf-4edf-a16f-ebcd54417fea.png)

some body can help me?

**Comment by fuchanghai** [comment]
@xizhimen may be you lost parameter
![image](https://user-images.githubusercontent.com/33984497/223042069-95a295b0-ccea-499d-966e-42b2d7101128.png)
